wagtail/contrib/redirects/utils.py

```diff
@@ -45,5 +47,10 @@ def get_file_storage():
     raise Exception("Invalid file storage, must be either 'tmp_file' or 'cache'")
 
 
+def get_redirect_hash(url):
```

Comment:
Non-essential suggestion: Might this be better off as a static method on the Redirect model? I feel it might make the whole process a little easier to reason about.

Reply:
I actually initially wrote it like that, can't remember why I switched it, must have been to follow the pattern on embeds. But I agree, having a static method works better, will push a commit with the change shortly.

Reply:
I'm thinking it was probably to have access to it in the data migration without having to import the model.